subjects/PE/x86/pySample/shingledPySample.reko/pySample_text.c

```diff
@@ -440,8 +440,8 @@ void fn10001864()
 	{
 		GetSystemTimeAsFileTime(fp - 0x0C);
 		ui32 esi_n = GetCurrentProcessId() ^ GetCurrentThreadId() ^ GetTickCount();
-		QueryPerformanceCounter(fp - 0x14);
-		ui32 esi_n = esi_n ^ (dwLoc10 ^ dwLoc14);
+		QueryPerformanceCounter(&tLoc14);
+		ui32 esi_n = esi_n ^ (tLoc14.dw0004 ^ tLoc14);
```

Comment:
What happened here? `tLoc14` seems to have sub-structure?

Reply:
Can we make an issue to track this then?

Reply:
Yes, we can.

Reply:
See #1159